src/parser/classes/ExpandableVideoDescriptionBody.ts

```diff
@@ -0,0 +1,21 @@
+import { YTNode } from '../helpers.js';
+import { type RawNode } from '../index.js';
+
+export default class ExpandableVideoDescriptionBody extends YTNode {
+  static type = 'ExpandableVideoDescriptionBody';
+
+  show_more_text: String;
+  show_less_text: String;
+  attributed_description_body_text: {
+    content: String
+  };
+
+  constructor(data: RawNode) {
+    super();
+    this.show_more_text = data.showMoreText.simpleText;
```

Comment:
These should be a `Text` object so we don't have to worry about YouTube suddenly using something else instead of `simpleText`